x-pack/solutions/observability/packages/utils_server/entities/get_data_streams_for_entity.ts

```diff
@@ -54,10 +54,26 @@ export async function getDataStreamsForEntity({
   });
 
   const dataStreams = uniq(
-    compact(await resolveIndexResponse.indices.flatMap((idx) => idx.data_stream))
+    compact(
+      await resolveIndexResponse.indices.flatMap((idx) => {
```

Comment:
Index names may include a remote cluster. Previously, when extracting the the data view, the remote cluster was not accounted for.

Reply:
I think the logic is broken anyway. What we should do is:

- get all the indices from `.indices`. if `data_stream` is defined, we use that - concatenating it with a potential remote cluster in `name`
- get all the data streams from `.data_streams` - these already have a remote cluster prefix

`GET _resolve/index/*:logs-activemq.audit-default*` will return an item in `data_streams`, while `GET GET _resolve/index/*:.ds-logs-activemq.audit-default*` will return items in `indices`, with `data_stream` being defined.